Game/RainMeadow.PlayerHooks.cs

```diff
@@ -95,6 +96,16 @@ private void Player_AddFood(On.Player.orig_AddFood orig, Player self, int add)
         }
     }
 
+    private int Player_FoodInRoom(On.Player.orig_FoodInRoom_bool orig, Player self, bool eatAndDestroy)
+    {
+        if (OnlineManager.lobby != null) {
```

Comment:
Check for storymode only. I don't think it'd effect other things, but just to be safe